packages/yourloops/pages/hcp/team-leave-dialog.tsx

```diff
@@ -0,0 +1,267 @@
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import * as React from "react";
+import { Trans } from "react-i18next";
+
+import { makeStyles, Theme } from "@material-ui/core/styles";
+import { red } from '@material-ui/core/colors';
+
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+
+import { Team } from "models/team";
+import { t } from "../../lib/language";
+import { useAuth } from "../../lib/auth/hook/use-auth";
+
+interface LeaveTeamDialogProps {
+  team: Team | null;
+  onShowModalLeaveTeam: (team: Team | null) => void;
+  onLeaveTeam: () => Promise<void>;
+}
+
+interface LeaveTeamDialogElementsProps {
+  team: Team | null;
+  teamName: string;
+  onlyMember: boolean;
+  userIsTheOnlyAdministrator: boolean;
+  buttonsDisabled: boolean;
+  handleClose: () => void;
+  handleLeaveTeam: () => Promise<void>;
+}
+
+const leaveTeamDialogClasses = makeStyles((theme: Theme) => {
+  return {
+    buttonCancel: {
+      marginRight: theme.spacing(2),
+    },
+    buttonLeaveAndDel: {
+      backgroundColor: red[500], // eslint-disable-line no-magic-numbers
+      '&:hover': {
+        backgroundColor: red[700], // eslint-disable-line no-magic-numbers
+      },
+    },
+  };
+});
+
+/**
+ * Return true if the userId is the only administrator of this team.
+ * @param team The team to test
+ * @param userId The current user id
+ */
+function isUserIsTheOnlyAdministrator(team: Team | null, userId: string): boolean {
+  if (team === null) {
+    return false;
+  }
+
+  const members = team.members ?? [];
+  let nAdmin = 0;
+  let userIsAdmin = false;
+  for (const member of members) {
+    if (member.role === "admin") {
+      nAdmin++;
+      if (member.userId === userId) {
+        userIsAdmin = true;
+      }
+    }
+  }
+  return userIsAdmin && nAdmin === 1;
+}
+
+function LeaveTeamDialogTitle(props: LeaveTeamDialogElementsProps): JSX.Element | null {
+  const { team, teamName, onlyMember, userIsTheOnlyAdministrator } = props;
+
+  if (team === null) {
+    return null;
+  }
+
+  if (onlyMember) {
+    return (
+      <DialogTitle id="team-leave-dialog-title">
+        <strong>{t("team-leave-dialog-and-del-title")}</strong>
+        <br />
+        <span id="team-leave-dialog-title-team-name">{teamName}</span>
+      </DialogTitle>
```

Comment:
This code is duplicated 3 times, the only difference is on the translation in the strong balise.
Maybe define a variable for it and return somthing like 
`
<DialogTitle id="team-leave-dialog-title">
        <strong>{t(myVariable}</strong>
        <br />
        <span id="team-leave-dialog-title-team-name">{teamName}</span>
 </DialogTitle>`

Reply:
A PR to fix all theses little things on the team screen will follow.